docs/src/modules/zend.cache.storage.adapter.rst

```diff
@@ -746,26 +746,26 @@ The Redis Adapter
-   |server              |                      |                         | Server can be described as follows:                                                  |
-   |                    |                      |                         |  - URI:   /path/to/sock.sock                                                         |
-   |                    |                      |                         |  - Assoc: array('host' => <host>[, 'port' => <port>[, 'timeout' => <timeout>]])      |
-   |                    |                      |                         |  - List:  array(<host>[, <port>, [, <timeout>]])                                     |
-   +--------------------+----------------------+-------------------------+--------------------------------------------------------------------------------------+
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |Name                |Data Type             |Default Value            |Description                                                                                   |
+   +====================+======================+=========================+==============================================================================================+
+   |database            |``integer``           |0                        |Set database identifier                                                                       |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |lib_option          |``array``             |``[]``                   | Associative array of redis options were the array key is the option name                     |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |namespace_separator |``string``            |":"                      |A separator for the namespace and prefix                                                      |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |password            |``string``            |""                       |Set password                                                                                  |
+   +--------------------+----------------------+-------------------------+----------------------------------------------------------------------------------------------+
+   |persistent_id       |``string``            |````                     |Set persistent id (*name* of the connection, [read more](http://redis.io/topics/persistence)) |
```

Comment:
Sure this is correct but you shouldn't link a documentation about something very different. Please only note that this `persistent_id` has nothing to do with persistence of the server.

PS: I'm also missing a note that an empty string will result in not using a persistent connection at all.